Code/Legacy/CryCommon/CryTypeInfo.cpp

```diff
@@ -131,8 +132,9 @@ const CTypeInfo&PtrTypeInfo()
 // bool
 AZStd::string ToString(bool const& val)
 {
-    static AZStd::string sTrue = "true", sFalse = "false";
-    return val ? sTrue : sFalse;
+    static AZStd::fixed_string<5> sTrue = "true";
+    static AZStd::fixed_string<6> sFalse = "false";
+    return val ? sTrue.c_str() : sFalse.c_str();
```

Comment:
You might might as well just return the literal string here.

The static variable isn't gain us any peformance.
```suggestion
    return val ? "true" :"false";
```